packages/gcp_vertexai/data_stream/logs/fields/fields.yml

```
@@ -0,0 +1,131 @@
+      type: flattened
+      description: "The operation request. This may not include all request elements, such as those that are too large, privacy-sensitive, or duplicated elsewhere in the log record. When the JSON object represented here has a proto equivalent, the proto name will be indicated in the @type property."
+    - name: response
+      type: flattened
+      description: "The operation response. This may not include all response elements, such as those that are too large, privacy-sensitive, or duplicated elsewhere in the log record. When the JSON object represented here has a proto equivalent, the proto name will be indicated in the @type property."
+    - name: status
+      type: group
+      fields:
+        - name: code
+          type: integer
+          description: "The status code, which should be an enum value of google.rpc.Code."
+        - name: message
+          type: keyword
+          description: "A developer-facing error message, which should be in English. Any user-facing  error message should be localized and sent in the google.rpc.Status.details  field, or localized by the client."
+        - name: details
+          type: flattened
+          description: |
+            A list of messages that carry the error details.
+    - name: policy_violation_info
+      type: group
+      fields:
+        - name: payload
+          type: flattened
+          description: Resource payload that is currently in scope and is subjected to orgpolicy conditions.
+        - name: resource_tags
+          type: flattened
+          description: Tags referenced on the resource at the time of evaluation.
+        - name: resource_type
+          type: keyword
+          description: Resource type that the orgpolicy is checked against.
+        - name: violations
+          type: nested
+          description: |
+            Provides information about the Policy violation info for the request.
+        #nested fields are handled this way due to https://github.com/elastic/integrations/pull/11016 
+        - name: violations.constraint
+          type: keyword
+          description: "Constraint name."
+        - name: violations.errorMessage
+          type: keyword
+          description: "Error message that policy is indicating."
+        - name: violations.checkedValue
+          type: keyword
+          description: "Value that is being checked for the policy."
+        - name: violations.policyType
+          type: keyword
+          description: "Indicates the type of the policy."
+    - name: authorization_info
+      type: nested
+      description: |
+        Authorization information for the operation.
+    - name: authorization_info.permission
+      type: keyword
+      description: "The required IAM permission."
+    - name: authorization_info.granted
+      type: boolean
+      description: "Whether or not authorization for resource and permission was granted."
+    - name: authorization_info.permissionType
```

rename this field to snake_case.
```suggestion
    - name: authorization_info.permission_type
```